bcdc_apitests/tests/resources/test_resources.py

```diff
@@ -114,7 +168,7 @@ def test_resource_delete(remote_api_admin_auth, resource_data):
 def test_package_delete(ckan_url, ckan_auth_header,
```

Comment:
This should be in a fixture, thinking that the test_resources.py should only include tests for resources.  package tests should be with the test_packages.py.  To delete I'd set up a module fixtures that creates the required package for the start of the resource test, and then when complete deletes the package.